### PR TITLE
Added POLL_DELAY environment variable to UserConfiguration class

### LINKED ISSUE #3413: replace POLL_DELAY with actual configuration
The environment variable `POLL_DELAY` is used as a sort of secret option to specify how long to wait before querying for jobs in the interactive sessions page. Let's replace this environment variable with an actual configuration folks can set.

Note that some sites may be using the environment variable - so we have to take care not to break that for them and just use precedence. 

https://github.com/search?q=repo%3AOSC%2Fondemand%20POLL_DELAY&type=code

### PR DESCRIPTION
**Pull request by abujeda**
Added POLL_DELAY environment variable to UserConfiguration class.

This change is backwards compatible with the environment variable.

Fixes #3413

### CODE REVIEW

**Comment by johrstrom** [comment]
I'm not sure if it should be a profile based thing. Generally folks want to control this so that `squeue` calls happen less frequently, so I'm not sure what the advantage here is to have one profile at 10 seconds and another at 20 for example.

**Comment by abujeda** [comment]
I always tend to add all properties to the UserConfiguration to make them available in profiles and, in this case, because it already supports ENV variables.

Will move it to the main Configuration object.

**Comment by abujeda** [comment]
Moved to Configuration object.

File: apps/dashboard/config/configuration_singleton.rb [hunk 1]
```diff
@@ -368,6 +368,11 @@ def ood_bc_card_time
     (ood_bc_card_time_int < 0) ? 0 : ood_bc_card_time_int
   end
 
+  def sessions_poll_delay
+    poll_delay = ENV['POLL_DELAY']
+    poll_delay.nil? ? config.fetch(:sessions_poll_delay, '10000') : poll_delay.to_s
+  end
```
Comment (johrstrom) [inline apps/dashboard/config/configuration_singleton.rb:374]: I have a couple things here.

First can we prepend `bc_` to this. That'll follow other configuration names so folks can see they all relate to batch_connect.

Secondly - I'm thinking we need some sort of check like casting `to_i` to be sure it's an integer. I also think that 10000 should be the default and likely minimum too. I'd hate to see someone configure `1` and just blow up their system lol.

Comment (abujeda) [reply apps/dashboard/config/configuration_singleton.rb]: Yes, that is better. I completed the updates

**Review by johrstrom — APPROVED** [review]
Thanks for the quick turnaround on this!

**Pull request merged**